### PR TITLE
Factorio

### PR DESCRIPTION
**Pull request by bkuhl**
* `php artisan integration:factorio-update` generates the list of unpublished releases and echoes them to console

### CODE REVIEW

File: app/Integrations/Factorio/ReleaseProvider.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                ($version->major() == 0 && $version->minor() < 14) ||
+                ($version->minor() == 14 && $startingExperimental->moreRecentThan($version)) ||
+                ($version->minor() == 15 && $startingStable->moreRecentThan($version))
+            ) {
+                continue;
+            }
+
+            // Factorio uses 0.15.2 instead of 0.15.02, so we prepend the zero and make it a number for
+            // simpler comparisons to determine order
+            $numericallyComparableVersion = $version->major().$version->minor().sprintf('%02d', $version->patch());
```
Comment (alexknutson) [inline app/Integrations/Factorio/ReleaseProvider.php]: nice 👍

File: database/seeds/DatabaseSeeder.php [hunk 2]
```diff
@@ -17,5 +18,10 @@ public function run()
         factory(User::class)->create();
 
         $this->call(ProductionSeeder::class);
+
+        //        factory(Release::class)->create([
```
Comment (alexknutson) [inline database/seeds/DatabaseSeeder.php]: Is this needed?

Comment (bkuhl) [inline database/seeds/DatabaseSeeder.php]: Has been removed

**Comment by bkuhl** [comment]
Still awaiting review

**Pull request merged**